com.vrcfury.vrcfury/Editor/VF/Feature/BakeHapticPlugsBuilder.cs

```diff
@@ -82,7 +82,10 @@ public class BakeHapticPlugsBuilder : FeatureBuilder {
                         tip.active = false;
                         var light = tip.AddComponent<Light>();
                         light.type = LightType.Point;
-                        light.color = Color.black;
+                        // Unity prioritises brighter lights when sorting unity_LightColor

```

Comment:
I don't believe this is actually the case. You can verify it yourself by going into the unity frame debugger while in play mode, but slot 4 is basically always broken in one way or another, and while adjusting light color can change the symptoms, it's not a fix, and giving this light a color will have other side effects.

Reply:
The comment might not describe the exact behaviour unity has

However, when dealing with pure black lights, adding a slight colour did cause the light to be sorted into slot 1, That was what I had observed using the unity frame debugger some time ago